### PR TITLE
porting po_instance table to metadb

### PR DESCRIPTION
**Pull request by natalyapik**
```
All queries:
- [ ] Pull request is based on a new branch (not main)
- [ ] Pull request scope is not overly broad
- [ ] Release notes have been added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

**Review by trapido — COMMENTED** [review]
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

This looks great! A couple of minor things I noticed:
1.I believe the metadb directive should be "--metadb:table po_instance"
2. Line 4 – “no source” – one quote is a double quote and one is a single quote
2. Line 49 and line 56 – there is an extra space before ::uuid
3. I am wondering why “billTo” is joined twice (line 54 and line 55) – is it specific to your institution’s data?
4.There is a typo in line 139: "Name od the purchase order" should be "Name of the purchase order"
5. Lines 123 and 141: "Weather" should be "Whether"

**Comment by natalyapik** [comment]
second join of config_data__t table are for "ship to" location, thank you for catching it

**Review by trapido — DISMISSED** [review]
Looks great!
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/po_instance.sql [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+COMMENT ON COLUMN po_instance.pol_location_id IS 'UUID of the location created for this purcase order line';
+
+COMMENT ON COLUMN po_instance.pol_location_name IS 'Name of the purchase order line location';
+
+COMMENT ON COLUMN po_instance.pol_location_source IS 'Wether location is a holdings location or permanent location of the purchase order line';
+
+COMMENT ON COLUMN po_instance.title IS 'Title of the material';
+
+COMMENT ON COLUMN po_instance.publication_date IS 'Date (year) of the material''s publication';
```
Comment (LauSchl) [inline sql_metadb/derived_tables/po_instance.sql:145]: Contains an additional " ' "

**Review by LauSchl — COMMENTED** [review]
Everything works great. Just a little typo :)
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/po_instance.sql (hunk 1 above)
Comment (amzoss) [reply sql_metadb/derived_tables/po_instance.sql:145]: @LauSchl: Just to confirm what @natalyapik is saying, this is actually the correct syntax for including a single quote inside a string in SQL. So, if this is the only concern, I think you can approve?

**Review by LauSchl — DISMISSED** [review]
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/po_instance.sql (hunk 1 above)
Comment (LauSchl) [reply sql_metadb/derived_tables/po_instance.sql:145]: Oh ok. My fault and sorry for the delay. I approved it now.

**Pull request merged**